src/main/java/lotto/LottoNumberBag.java

```diff
@@ -1,19 +1,23 @@
 package lotto;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import static lotto.LottoNumberGenerator.LOTTO_NUMBER_SIZE;
 
 public class LottoNumberBag implements NumberBag {
 
+    static final String SPLIT_DELIMITER = ",";
+
```

Comment:
아주아주 사소한 부분이지만 실수하신 것 같아 코멘트만 남기겠습니다 😁

```suggestion
    private static final String SPLIT_DELIMITER = ",";
```

Reply:
꼼꼼히 봐주셔서 감사합니다 !! 🙇🏻‍♂️
이런 사소한 실수를 없애고 싶은데요 😭 
꼼꼼하게 봐주신 덕에 저도 한번 더, 두번 더 확인해보는 습관을 가질 수 있을 것 같아요 !! 
감사합니다 ! 😊